packages/broker/src/config/ConfigWizard.ts

```diff
@@ -1,254 +1,544 @@
+import path from 'path'
+import { z } from 'zod'
+import {
+    CURRENT_CONFIGURATION_VERSION,
+    formSchemaUrl,
+} from '../config/migration'
+import { generateMnemonicFromAddress } from '../helpers/generateMnemonicFromAddress'
 import * as MqttConfigSchema from '../plugins/mqtt/config.schema.json'
+import * as WebsocketConfigSchema from '../plugins/websocket/config.schema.json'
 import * as BrokerConfigSchema from './config.schema.json'
-import { getDefaultFile } from './config'
-import { CURRENT_CONFIGURATION_VERSION, formSchemaUrl } from '../config/migration'
-import { generateMnemonicFromAddress } from '../helpers/generateMnemonicFromAddress'
-import { toEthereumAddress } from '@streamr/utils'
+import { ConfigFile, getDefaultFile } from './config'
 
-export interface PrivateKeyAnswers extends Answers {
-    generateOrImportPrivateKey: 'Import' | 'Generate'
-    importPrivateKey?: string
-}
+const MinBalance = utils.parseEther('0.1')
+
+export const start = async (): Promise<void> => {
+    log(`
+        >
+        > ***Welcome to the Streamr Network!***
+        > This Config Wizard will help you setup your node.
+        >
+        > The steps are documented here:
+        > *https://docs.streamr.network/guides/how-to-run-streamr-node#config-wizard*
+        >
+    `)
+
+    try {
+        const privateKey = await getPrivateKey()
+
+        const nodeAddress = new Wallet(privateKey).address
+
+        const network = await getNetwork()
+
+        const operator = await getOperatorAddress()
+
+        const operatorPlugins = operator ? {
+            operator: {
+                operatorContractAddress: operator,
+            },
+        } : {}
+
+        const { http, ...pubsubPlugins } = await getPubsubPlugins()
+
+        if (http) {
+            Object.assign(pubsubPlugins, {
+                http: {},
+            })
+        }
+
+        const storagePath = await getStoragePath()
+
+        const httpServer = http?.port ? { port: http.port } : void 0
+
+        const config: ConfigFile = {
+            $schema: formSchemaUrl(CURRENT_CONFIGURATION_VERSION),
+            client: {
+                auth: {
+                    privateKey,
+                },
+            },
+            plugins: {
+                ...operatorPlugins,
+                ...pubsubPlugins,
+            },
+            httpServer,
+        }
+
+        persistConfig(
+            storagePath,
+            network === 'polygon' ? config : getMumbaiConfig(config)
+        )
+
+        log(`
+            >
+            > ~ *Congratulations, you've setup your Streamr node!*
+            > Your node address is ${chalk.greenBright(nodeAddress)}
+            > Your node's generated name is ${chalk.greenBright(getNodeMnemonic(privateKey))}
+        `)
+
+        if (operator) {
+            const resume = progress((f) =>
+                style(`> Your node address has *${f} MATIC* _– checking balance…_`)
+            )
+
+            try {
```

Comment:
Why is this additional formating needed? Is it for removing `ETH` word?

Reply:
`progress` prints out a busyness indicator while we're checking the wallet balance. This entire line gets erased and replaced with `Your node address has *12.34 MATIC* (…)` once we know the value.

Both texts are formatted the same way and emphasise the important part.

Reply:
Refered to `.replace(/\.(\d+)/, (f) => f.substring(0, 3))` snippet (maybe that line was the line `92` at the time of comment, or something)